internal/pkg/template/templates/workloads/services/rd-web/manifest.yml

```
@@ -19,24 +19,40 @@ image:
 {{- end}}
   # Port exposed through your container to route traffic to it.
   port: {{.ImageConfig.Port}}
-
+{{if and .Private.IsBasic .Private.Basic}}
```

wut 💭 this is strange to read, why not:
```suggestion
{{if  .Private.IsBasic }}
```